app/models/pipeline_run.rb

```diff
@@ -1061,6 +1061,42 @@ def compare_ercc_counts
     ret
   end
 
+  def outputs_by_step
+    return unless pipeline_run_stages
```

Comment:
is this line necessary? if there are no pipeline_run_stages then the block below just won't run anything right? when might there be files returned by sample.result_folder_files but no pipeline_run_stages? and in that case, would we  want to show nothing at all?

Reply:
agreed, I removed the line